layouts/partials/flex/body-beforecontent.html

```diff
@@ -1,58 +1,87 @@
 <header>
-
-  <div class="logo">
-    {{ partial "header.html" . }}
-  </div>
+  <div class="contentWidth">
+    <div class="logo">
+      {{ partial "header.html" . }}
+    </div>
 
 
 
-  <div class="burger"><a href="javascript:void(0);" style="font-size:15px;">&#9776;</a></div>
-    <nav class="shortcuts">
+    <div class="burger"><a href="javascript:void(0);" style="font-size:15px;">&#9776;</a></div>
+      <nav class="shortcuts">
         {{- partial "search.html" . }}
-        <div title="v{{.Site.Data.Solo.CodeVersion}}"
-             style="overflow: hidden;
-             text-overflow: ellipsis;
-             white-space: nowrap;"
-        >v{{ .Site.Data.Solo.CodeVersion }}</div>
-    {{- with .Site.Menus.shortcuts}}
-        {{- range sort . "Weight"}}
+
+        <!--<div title="v{{.Site.Data.Solo.CodeVersion}}"
+            style="overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;"
+        >
+          {{ .Site.Data.Solo.CodeVersion }}
+        </div>-->
+        <label for="version-selection">Version:</label>
+        <select id="version-selection" onchange="javascript:location.href = this.value;">
+          <option value="/latest/{{ .RelPermalink}}" {{if in .RelPermalink "latest/"}} selected{{end}}>
+            latest (1.3.0)
+          </option>
+          <option value="/1.2.0/{{ .RelPermalink}}" {{if in .RelPermalink "1.2.0/"}} selected{{end}}>
+            1.2.0
+          </option>
+          <option value="/1.1.0/{{ .RelPermalink}}" {{if in .RelPermalink "1.1.0/"}} selected{{end}}>
+            1.1.0
+          </option>
+          <option value="/1.0.0/{{ .RelPermalink}}" {{if in .RelPermalink "1.0.0/"}} selected{{end}}>
+            1.0.0
+          </option>
+        </select>
```

Comment:
these are the currently hosted live versions: https://github.com/solo-io/docs-prod/blob/master/cluster/docs-virtualservice.yaml#L31-L103

latest will track the master branch, which is currently at 1.3.2. we should also add tabs for the other missing versions